src/js/components/Accordion/__tests__/Accordion-test.tsx

```diff
@@ -26,20 +27,20 @@ describe('Accordion', () => {
 
     const results = await axe(container);
     expect(results).toHaveNoViolations();
-    expect(container).toMatchSnapshot();
+    expect(asFragment()).toMatchSnapshot();
   });
 
   test('no AccordionPanel', () => {
-    const { container } = render(
+    const { asFragment } = render(
```

Comment:
can `asFragment` come from "screen"? Or it needs to get defined in this way?

Reply:
Only queries (`getByRole`, `getByLabelText`, etc.) can come from screen so `asFragment` can't come from screen and has to be defined like that

Reply:
Thanks!